doc/changes/changes_0.1.0.md

```diff
@@ -1,10 +1,15 @@
-#cloudwatch-adapter 0.1.0, released 2021-03-02
+# CloudWatch-adapter 0.1.0, released 2021-03-03
 
-## Code name: Initial release
+Code name: Initial release
 
 ## Summary
 
-This is the very first release of the Exasol – Cloud watch adapter.
+This is the very first release of the Exasol – CloudWatch adapter.
+
+Known issue:
```

Comment:
I think you can add one sentence here, to check the known issues in the known bugs section.